grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
Just to clarify - I'd be more concerned with us doing something like this as a matter of habit, rather than with this particular occurrence.  I think we need to be very prudent with our use of feature toggles - especially w.r.t. to making them short lived, as @danfruehauf eludes to.